plugins/azure-devops-common/api-report.md

```diff
@@ -144,6 +146,7 @@ export enum PolicyTypeId {
   MinimumReviewers = 'fa4e907d-c16b-4a4c-9dfa-4906e5d171dd',
   RequiredReviewers = 'fd2167ab-b0be-447a-8ec8-39368250530e',
   Status = 'cbdc66da-9728-4af8-aada-9a5a32e4a226',
+  WorkItem = '40e92b44-2fe1-4dd6-b3d8-74a9c21d0c6e',
```

Comment:
I've double checked: this GUID is correct and matches the Policy Type listed. I do think there is some more changes needed. You'll need to add the policy to this switch: https://github.com/backstage/backstage/blob/e365e00aca79a97e881ad4d5317b1c77b8d93b56/plugins/azure-devops-backend/src/utils/azure-devops-utils.ts#L140 otherwise adding the Policy Type doesn't do anything from what I can tell

Reply:
@awanlin Thank you for bringing this to my attention. Will fix.